runtime/src/main/java/org/corfudb/runtime/view/AbstractOrderedGUIDFactory.java

```diff
@@ -0,0 +1,18 @@
+package org.corfudb.runtime.view;
+
+/**
+ * Globally Unique Identity (GUID) generator that returns compact GUIDs having a notion of
+ * a comparable global ordering.
+ *
+ * Why not use Java UUIDs?
+ *   JAVA UUIDs lack cross node comparability or global ordering in the absence of time sync.
+ *   They are also 36 bytes long which can be an overkill for simple use cases.
```

Comment:
hmm, are they really?

UUID constructor uses two longs. In java a long is only 8 bytes. 

```java
public UUID(long mostSigBits, long leastSigBits) {
        this.mostSigBits = mostSigBits;
        this.leastSigBits = leastSigBits;
    }
```

Reply:
Good point. Only the string representation of the UUID is 36 bytes. They do lack global ordering though. I will update the comment. Thanks!